packages/schema/turbo.json

```diff
@@ -1,19 +1,52 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "extends": ["//"],
+  "extends": [
+    "//"
+  ],
   "pipeline": {
-    "prep": {
-      "dependsOn": ["@custom/cms#prep:composer"],
+    "prep:codegen": {
+      "dependsOn": [
+        "@custom/cms#prep:composer"
+      ],
       "inputs": [
-        "../../apps/website/node_modules/@amazeelabs/*/directives.graphql",
-        "../../apps/cms/web/modules/contrib/*/directives.graphql",
-        "../../apps/cms/web/modules/custom/*/directives.graphql",
-        "src/**",
```

Comment:
Aren't we still depending on these external schema bits?

Reply:
Good question. Yes, we do. But they are outputs of `@custom/cms#prep:composer`, which we depend on here. So that should be fine, no?

Reply:
I think this is a leftover from times where composer was not covered by turbo.

Reply:
We'll see if it works :)

Reply:
Actually not true! 😄 
For composer stuff, maybe, but directives in custom modules still need to be depended on.

Reply:
Which should be in the central `schema.graphql` anyway ...